SPECS/ocaml-base/ocaml-base.spec

```diff
@@ -52,15 +48,15 @@ The %{name}-devel package contains libraries and signature files for
 developing applications that use %{name}.
 
 %prep
-%autosetup -n base-%{version} -p1
+%autosetup -n %{srcname}-%{version} -p1
 
 %build
 %dune_build
 
 %install
 %dune_install
 
-%if %{with test}
+%if %{with_check}
```

Comment:
You can drop `%if %{with_check}` and `%endif` completely. The check is only ran during tests builds anyway.